CONTRIBUTING.md

```diff
@@ -0,0 +1,48 @@
+# Contributing to redux-saga
+Contributing to redux-saga isn't limited to just filing bugs, users are more than welcomed to make suggestions, report any issue they may find, and make pull requests to help better redux-saga.
+
+## Filing a bug
+When filing a bug, please follow the provided template:
+
+```
+### Description of the bug/issue
+
+# Steps to reproduce the bug/issue
+
+1.
+2.
+3.
+...
+
+# Your results
+
+# The expected reults
+
+# System information
+```
+
+## Working on redux-saga
+### Prerequisites
+* [Git](https://git-scm.com/)
+* Latest version of [Node.js](https://nodejs.org/en/)
+
+### Getting redux-saga
+1. Fork a copy of our repo
+2. Open up Git in an environment of your choice
+3. Run the following
+
+```
+$ git clone https://github.com/YOUR-GITHUB-PROFILE-NAME/redux-saga.git
+$ cd redux-saga
+$ npm install

```

Comment:
actually this should be `npm run bootstrap` now as we've started using lerna which manage this monorepo